src/tribler-gui/tribler_gui/widgets/channelcontentswidget.py

```diff
@@ -268,7 +270,13 @@ def go_back(self, checked=False):
     def on_breadcrumb_clicked(self, tgt_level):
         if int(tgt_level) + 1 != len(self.channels_stack):
             self.go_back_to_level(tgt_level)
+        elif isinstance(self.model, SearchResultsModel) and len(self.channels_stack) == 1:
+            # In case of remote search, when only the search results are on the stack,
```

Comment:
**praise:** you add an explanation to the code blocks, it is great!

Reply:
If something was pointed out by a colleague as unobvious, that should be commented :wink: